jsx/behavioural_feedback_panel.jsx

```diff
@@ -0,0 +1,606 @@
+var SliderPanel = React.createClass({
```

Comment:
this is just jsx so the formatting isn't that important (as the js file is what is use) but i will run it through a linter.